FIAT/barycentric_interpolation.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2015 Imperial College London and others.
```

Comment:
This is wrong. I think, if anything, this should be "Copyright Pablo Brubeck". Or we can just leave it out completely.